### PR TITLE
feat: add TitleBar, Visualization and StartScreen components

### PR DESCRIPTION
**Pull request by edoardo**
TitleBar is a simple connected component that shows the AO title

Visualization is a connected component that fires the analytics requests and renders the data in the response in a `DataTable`.
Screenshot 2021-10-06 at 13 53 47

StartScreen is a simple connected component that shows the last viewed AOs. Proper texts are TBD by @cooper-joe 
image

### CODE REVIEW

File: src/actions/user.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { apiFetchUserAuthority } from '../api/user'
+import { GenericServerError } from '../modules/error'
+import { RECEIVED_USER, SET_USER_AUTHORITY } from '../reducers/user'
+import { acSetLoadError } from './loader'
+
+export const acReceivedUser = value => ({
```
Comment (jenniferarnesen) [inline src/actions/user.js:6]: I think that using the word "Received" is possibly a leftover from when we were just learning how to implement redux. (at least I can remember following a tutorial that used that word). Wondering if we want to move away from it and use SET/GET?

File: src/components/Visualization/Visualization.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+            setAnalyticsResponse(analyticsResponse)
+        }
+
+        doFetch()
+    }, [visualization, page, pageSize, sortField, sortDirection])
+
+    useEffect(() => {
+        if (analyticsResponse) {
+            console.log('res', analyticsResponse)
```
Comment (martinkrulltott) [inline src/components/Visualization/Visualization.js:76]: remove console.log?

File: src/components/Visualization/Visualization.js [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+            setHeaders(headers)
+
+            // extract rows
+            setRows(
+                analyticsResponse.rows.reduce((filteredRows, row) => {
+                    filteredRows.push(
+                        headers.reduce((filteredRow, header) => {
+                            filteredRow.push(
+                                header ? row[header.getIndex()] : '-'
+                            ) // FIXME
```
Comment (martinkrulltott) [inline src/components/Visualization/Visualization.js:93]: FIXME what?

File: src/components/Visualization/styles/StartScreen.module.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+}
+.title {
+    font-weight: bold;
+    margin-top: 0;
+    line-height: 22px;
+    color: var(--colors-grey700);
+}
+.section {
+    text-align: left;
+    background: #fff;
```
Comment (martinkrulltott) [inline src/components/Visualization/styles/StartScreen.module.css:51]:
```suggestion
    background: var(--colors-white);
```

File: src/reducers/loader.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+            }
+        default:
+            return state
+    }
+}
+
+// Selectors
+export const sGetLoadError = state => state.loader.loadingError
+export const sGetIsLoading = state => state.loader.isLoading
+export const sGetIsPluginLoading = state => state.loader.isPluginLoading
```
Comment (martinkrulltott) [inline src/reducers/loader.js:43]: rename to something more generic? since there's no plugin now...

**Review by martinkrulltott — APPROVED** [review]

**Pull request merged**